aries-site/src/examples/templates/forms/PayExample.js

```diff
@@ -105,6 +105,19 @@ const FormContainer = ({ ...rest }) => {
   );
 };
 
+const Error = ({ textError, ...rest }) => {
+  return (
+    <Box {...rest} align="center" gap="xsmall" direction="row">
+      <CircleAlert size="small" />
+      <Text size="xsmall">{textError}</Text>
+    </Box>
+  );
+};
```

Comment:
Related to: https://github.com/hpe-design/design-system/pull/999/files#r463792055

```suggestion
const Error = ({ children, ...rest }) => {
  return (
    <Box align="center" background="background-front" gap="xsmall" direction="row"  {...rest} >
      <CircleAlert size="small" />
      <Text size="xsmall">{children}</Text>
    </Box>
  );
};
```